CMakeLists.txt

```diff
@@ -34,6 +34,9 @@ include(leatherman)
 # Add other dependencies
 find_package(Boost 1.54 REQUIRED COMPONENTS thread date_time chrono system program_options)
 
+# Add nanosecond resolution to boost::posix_time::time_duration
+add_definitions(-DBOOST_DATE_TIME_POSIX_TIME_STD_CONFIG)
```

Comment:
I have concerns about this as a library. I think we need to document it in config.hpp and the README, and define in config.hpp for those consuming hocon as a library.

If it becomes a problem for someone, we can later make this a compile-time flag and have split implementation depending on the flag.